jsk_2016_01_baxter_apc/launch/bimanual_stow_data_collection.launch

```diff
@@ -0,0 +1,200 @@
+
+    <node name="closing_mask_rgb"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_perception/Closing $(arg manager)">
+      <remap from="~input" to="/pi_to_mask_rgb/output" />
+      <rosparam>
+        size: 8
+        iterations: 4
+      </rosparam>
+    </node>
+
+    <node name="apply_mask_rgb"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_perception/ApplyMaskImage $(arg manager)">
+      <remap from="~input" to="/$(arg opposite_hand)_hand_camera/rgb/image_raw" />
+      <remap from="~input/mask" to="/closing_mask_rgb/output" />
+      <rosparam>
+        approximate_sync: true
+        queue_size: 1000
+      </rosparam>
+    </node>
+
+    <node name="in_hand_clipper_depth"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_pcl/AttentionClipper $(arg manager)">
+      <remap from="~input/points" to="/$(arg opposite_hand)_hand_camera/depth/points" />
+      <rosparam subst_value="true">
+        dimension_x: 0.3
+        dimension_y: 0.3
+        dimension_z: 0.2
+        initial_pos: [0.0, 0.0, 0.15]
+        initial_rot: [0, 0, 0]
+        frame_id: $(arg hand)_gripper_vacuum_pad
+      </rosparam>
+    </node>
+
+    <node name="pi_to_mask_depth"
+        type="nodelet" pkg="nodelet"
+        args="load jsk_pcl/PointIndicesToMaskImage $(arg manager)">
+      <remap from="~input" to="/in_hand_clipper_depth/output/point_indices" />
+      <remap from="~input/image" to="/$(arg opposite_hand)_hand_camera/depth/image_raw" />
+      <rosparam>
+        approximate_sync: true
+        queue_size: 1000
+      </rosparam>
+    </node>
+
+    <node name="closing_mask_depth"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_perception/Closing $(arg manager)">
+      <remap from="~input" to="/pi_to_mask_depth/output" />
+      <rosparam>
+        size: 8
+        iterations: 4
+      </rosparam>
+    </node>
+
+    <node name="apply_mask_depth"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_perception/ApplyMaskImage $(arg manager)">
+      <remap from="~input" to="/$(arg opposite_hand)_hand_camera/depth/image_raw" />
+      <remap from="~input/mask" to="/closing_mask_depth/output" />
+      <rosparam>
+        approximate_sync: true
+        queue_size: 1000
+      </rosparam>
+    </node>
+
+    <!-- for gripper pose -->
+    <node name="gripper_transform_publisher"
+        pkg="jsk_2016_01_baxter_apc" type="transform_publisher.py">
+        <rosparam subst_value="true">
+          duration: 1.0
+          parent_frame_id: /base
+          child_frame_id: /$(arg hand)_gripper_vacuum_pad
+        </rosparam>
+    </node>
+
+    <!-- for camera pose -->
+    <node name="camera_transform_publisher"
+        pkg="jsk_2016_01_baxter_apc" type="transform_publisher.py">
+        <rosparam subst_value="true">
+          duration: 1.0
+          parent_frame_id: /base
+          child_frame_id: /$(arg opposite_hand)_hand_camera_rgb_optical_frame
+        </rosparam>
+    </node>
```

Comment:
I'm not sure why you don't save tf from camera to vacuum_pad.
Because I thought you're creating vision-based system which is not Baxter specific.
If this is not robot-specific, saving tf from `/base` is not reasonable for me.